app/src/common/shared/com/igalia/wolvic/VRBrowserActivity.java

```diff
@@ -472,9 +471,7 @@ private void showWhatsNewDialogIfNeeded() {
         if (SettingsStore.getInstance(this).isWhatsNewDisplayed()) {
             return;
         }
-        if (shouldOpenInKioskMode(getIntent())) {
-            return;
-        }
+        // TODO do not show in kiosk mode?
```

Comment:
Not sure what this is about

Reply:
This is about not showing the What's New dialog in kiosk mode.

I have replaced that check with `mWindows.getFocusedWindow().isKioskMode()`.